### DISCUSSION TITLE
Dashboard显示Server Error (500)？

### DISCUSSION THREAD

**Discussion by a8159236** [hhyo/Archery#2276] [General]
![image](https://github.com/hhyo/Archery/assets/32159630/bd379b28-1ddb-465e-8cde-58c96150c0b1)
版本信息
应用版本/分支：Release v1.10.0
部署方式：Docker
请大佬指教

**Comment by a8159236** [comment]
已解决，redis连接失败导致。

**Reply by ashermed-lhc** [reply]
我也是这个报错你怎么解决的，我是helm安装

**Reply by ashermed-lhc** [reply]
我连外部的redis是OK的，我想连内部启动的redis，他默认启的是个集群，他连的时候感觉是连一个点的，所以报错
